### DISCUSSION TITLE
Migration 2.5.X - RC 3.0.X - Brush Name for Hue Brush

### DISCUSSION THREAD

**Discussion by Tenjim** [AvaloniaCommunity/Material.Avalonia#173] [Q&A] [answered]
Hi Guys,

I'm migrating a 2.5.X nightly build to RC 3.0.X and it's not very clear to me. 
In wiki, you said that all Swatch and Hue will be removed in 3.0.X Material.Avalonia. So I'm looking to replace these brush names :

```
PrimaryHueLightBrush
PrimaryHueLightForegroundBrush
PrimaryHueMidBrush
PrimaryHueMidForegroundBrush
PrimaryHueDarkBrush
PrimaryHueDarkForegroundBrush

SecondaryHueLightBrush
SecondaryHueLightForegroundBrush
SecondaryHueMidBrush
SecondaryHueMidForegroundBrush
SecondaryHueDarkBrush
SecondaryHueDarkForegroundBrush
```

Can you confirm these brush name will be erase in the future version ? (Maybe I misunderstood something about migration tips).

Thanks ! :)

**Comment by SKProCH** [comment, marked as answer]
Hello, @Tenjim 

No, this is misunderstanding. We removing `Hue` and `Swatch` C# classes. As for the brushes you listed - they will not change in any way and will be available.
